src/bin/console/console.cpp

```diff
@@ -604,17 +607,28 @@ int Console::_export_table(const std::string& args) {
   const std::string& filepath = arguments.at(1);
 
   const auto& storage_manager = Hyrise::get().storage_manager;
-  if (!storage_manager.has_table(tablename)) {
-    out("Error: Table does not exist in StorageManager\n");
-    return ReturnCode::Error;
+  const auto& meta_table_manager = Hyrise::get().meta_table_manager;
+
+  std::shared_ptr<AbstractOperator> table_operator = nullptr;
+  if (std::string_view(tablename).starts_with(MetaTableManager::META_PREFIX)) {
+    if (!meta_table_manager.has_table(tablename)) {
+      out("Error: MetaTable does not exist in MetaTableManager\n");
+      return ReturnCode::Error;
+    }
+    table_operator = std::make_shared<TableWrapper>(meta_table_manager.generate_table(tablename));
+  } else {
+    if (!storage_manager.has_table(tablename)) {
+      out("Error: Table does not exist in StorageManager\n");
+      return ReturnCode::Error;
+    }
+    table_operator = std::make_shared<GetTable>(tablename);
   }
 
   out("Exporting \"" + tablename + "\" into \"" + filepath + "\" ...\n");
-  auto get_table = std::make_shared<GetTable>(tablename);
-  get_table->execute();
+  table_operator->execute();
```

Comment:
I'd move this above the `out`, as it isn't really part of the exporting / filepath step